.gitignore

```diff
@@ -173,4 +173,6 @@ tags
 # ruff
 .ruff_cache
 
-wandb
\ No newline at end of file
+wandb
+
+src/controlnet_aux/main.py
```

Comment:
This is probably fine but a little unexpected?

Reply:
Yes, it was from local testing